test/integration/reconciliation.test.js

```diff
@@ -66,7 +66,7 @@ describe('Reconciliation', function() {
     expect(repo).toHaveState('count', 6)
   })
 
-  it('archived actions are removed from the archive', function() {
+  it.dev('archived actions are removed from the archive', function() {
```

Comment:
mean to leave this test as `.dev` ?

Reply:
Yeah. I should pick a better name. `.dev` means "only run this in the development test command". Microcosm also tests against the production build, but that doesn't have any of the assertions of the development version.

So this lets me test both dev and prod; taking advantage of assertions while also making sure the production build works.